newIDE/app/scripts/theme-templates/theme.json

```diff
@@ -309,6 +309,14 @@
         }
       }
     },
+    "event-drop-indicator": {
```

Comment:
Should this be the same as the fields you added in the different themes? `drop-indicator`, `cannot-drop`, `can-drop`, etc.

Reply:
Indeed, forgot that one !